build.rs

```diff
@@ -23,7 +23,6 @@
 /// ```
 use chrono::Utc;
```

Comment:
Were the changes in this file provoked by `clippy`? The reason I'm asking is that I thought `master` was clippy-compliant so I'm afraid there might be some kind of version mismatch.

Reply:
Master was not clippy compliant yet, no. I wanted to do so but thought about opening a new PR but might as well do it now

Reply:
![image](https://user-images.githubusercontent.com/3048875/103882482-c5b18880-50d3-11eb-98fb-f613358e3373.png)

It seems we might be getting different results. What are we doing differently?

Reply:
Clippy has this weird behavior (I've experienced it in other projects) where it reports refactors and other suggestions only when fresh building.

Try `cargo clean` followed by the `cargo clippy --all-targets`

Reply:
![image](https://user-images.githubusercontent.com/16060539/103884053-f1ce0900-50d5-11eb-9121-107def123a81.png)
![image](https://user-images.githubusercontent.com/16060539/103884067-f5619000-50d5-11eb-8ae6-4d296b8ce82f.png)
![image](https://user-images.githubusercontent.com/16060539/103884071-f7c3ea00-50d5-11eb-9236-bc9a10bbdf37.png)

Reply:
Worth noting I have `rustc` and `cargo` version `1.49` . I tested now with your version (`1.48`) and I still get the same suggestions.

Reply:
After `crate clean` I got the same result. When the same kind of thing happens in the future please open a separate PR. Keeping it 1 PR <=> 1 responsibility is very important as it not only eases the review process (for both parties) as it generates isolated changes (commits).

In this case, it would require open a PR on top of master and then builds this PR on top of that branch.